integ/src/main.rs

```diff
@@ -197,6 +198,10 @@ async fn run() -> Result<()> {
                 .context(error::RunBrupopSnafu)?;
             tokio::time::sleep(WAIT_CERT_MANAGER_COMPLETE).await;
             info!("Running brupop on existing EKS cluster ...");
+            if let Err(error) = process_brupop_namespace(Action::Create, &kube_config_path).await {
+                warn!("Error creating brupop namespace: {:?}", error)
+            };
```

Comment:
Should we exit here? I could see this being non-fatal if you get this error when such a namespace already exists, but otherwise it seems like we should return an error here.

Reply:
> I could see this being non-fatal if you get this error when such a namespace already exists

Exactly. I ran into a few cases where the integration tests didn't completely remove the namespace (or there were hanging resources preventing the namespace from being purged, my temporary cluster creds timed-out, etc. etc.) and it was just easier to warn instead of completely bail out of creation.

But if we think this is too loose, I can also see exiting early here and requiring the operator to fix the problem.

Reply:
Circling back on this - ended up removing `process_brupop_namespace` since the final generated yaml manifest has the namespace resource 👍🏼